docs/workflows.md

```diff
@@ -1,39 +1,41 @@
-# Bevy Quickstart CI / CD
+# Bevy Quickstart Workflows
 
-This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) for [CI / CD](https://www.redhat.com/en/topics/devops/what-is-ci-cd) (defined in [`.github/workflows/`](../.github/workflows)).
+This template uses [GitHub workflows](https://docs.github.com/en/actions/using-workflows) for [CI / CD](https://www.redhat.com/en/topics/devops/what-is-ci-cd).
+They are defined in [`.github/workflows/`](../.github/workflows).
 
 ## CI (testing)
 
-The [CI workflow](.github/workflows/ci.yaml) will trigger on every commit to `main`, and:
+The [CI workflow](.github/workflows/ci.yaml) will trigger on every commit or PR to `main`, and does the following.
 
 - Run tests.
 - Run Clippy lints.
 - Check formatting.
 - Check documentation.
 
 > [!Tip]
-> You may want to set up a [GitHub ruleset](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets) to require that all commits to `main` pass CI.
+> You may want to set up a [GitHub ruleset](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets) to require that all commits to `main` pass CI. The following picture shows how such a setup would look like:
+> ![A screenshot showing a GitHub ruleset with status checks enabled](workflow-ruleset.png)
 
 ## CD (releasing)
 
 The [CD workflow](../.github/workflows/release.yaml) will trigger on every pushed tag in the format `v1.2.3`, and:
 
-- Create a release build for Windows, MacOS, Linux, and web.
+- Create a release build for Windows, macOS, Linux, and web.
 - (Optional) Upload to [GitHub releases](https://docs.github.com/en/repositories/releasing-projects-on-github).
 - (Optional) Upload to [itch.io](https://itch.io).
 
 <details>
   <summary>This workflow can also be triggered manually.</summary>
-    
+
 In your GitHub repository, navigate to `Actions > Release > Run workflow`:
 
-![UI demonstration](./release-workflow-dispatch.png)
+![A screenshot showing a manually triggered workflow on GitHub Actions](./workflow-dispatch-release.png)
```

Comment:
The accessibility of the alt tags is not perfect, but I made them a bit better.
See https://www.w3.org/WAI/tutorials/images/informative/